cla_backend/libs/eligibility_calculator/tests/test_calculator.py

```diff
@@ -10,7 +10,28 @@
 from ..models import CaseData, Facts
 
 
-class TestCapitalCalculator(unittest.TestCase):
+class MortgageCapRemovalMixin(object):
+    def setUp(self):
+        super(MortgageCapRemovalMixin, self).setUp()
+        super(MortgageCapRemovalMixin, self).setUp()
```

Comment:
Please tell me this duplication is just a mistake and not needed for some horrible reason... 😅

Reply:
Good catch. Yep it's a mistake